doc/whats_new/v0.24.rst

```diff
@@ -48,15 +48,20 @@ Changelog
 :mod:`sklearn.datasets`
 .......................
 
+- |Enhancement| :func:`datasets.fetch_covtype` now now supports the optional
+  argument `as_frame`; when it is set to True, the returned Bunch object's
+  `data` and `frame` members are pandas DataFrames, and the `target` member is
+  a pandas Series. :pr:`17491` by :user:`Alex Liang <tianchuliang>`.
+
 - |Enhancement| :func:`datasets.fetch_openml` now allows argument `as_frame`
   to be 'auto', which tries to convert returned data to pandas DataFrame
   unless data is sparse.
   :pr:`17396` by :user:`Jiaxiang <fujiaxiang>`.
 
-- |Enhancement| :func:`datasets.fetch_covtype` now now supports the optional
-  argument `as_frame`; when it is set to True, the returned Bunch object's
-  `data` and `frame` members are pandas DataFrames, and the `target` member is
-  a pandas Series. :pr:`17491` by :user:`Alex Liang <tianchuliang>`.
+- |API| The default value of `as_frame` in :func:`datasets.fetch_openml` will
+  change from False to 'auto' in 0.25. It now issues a `FutureWarning` when
```

Comment:
Is it really 0.25? I would have thought that this change will get into 0.24 so 2 major release policy would means that we can change the default in 0.26?

Reply:
We have a note that says that `fetch_opeml` is experimental so usually, we use one cycle for this case. And the Note is explicitly about returned type :)